spec/vcr_helper.rb

```diff
@@ -3,16 +3,47 @@
 VCR.configure do |c|
   c.cassette_library_dir = 'vcr/cassettes'
   c.hook_into :webmock
-  c.configure_rspec_metadata!
   c.default_cassette_options = {
     erb: true,
     exclusive: true,
     allow_playback_repeats: true,
     match_requests_on: [:method, VCR.request_matchers.uri_without_param(:api_key)]
   }
 
-  # Do not capture requests other than to the API endpoints
+  # Enable VCR logging using, for example,
+  #  `VCR_DEBUG=1 rspec ./spec/controllers/external_users/claims_controller_spec.rb`
+  c.debug_logger = File.open(c.cassette_library_dir + '/vcr_debug.log', 'w') if ENV['VCR_DEBUG']
+
+  # Ignore requests other than to the API endpoints and LAA fee calculator
   c.ignore_request do |request|
     !URI(request.uri).path.start_with?('/api/')
   end
 end
+
+# use `VCR_OFF=true rspec` too turn off vcr
+VCR.turn_off! if ENV['VCR_OFF']
+
+# custom VCR request matcher to match request based on
+# path and query but not host because laa-fee-calculator
+# host could change and responses are path and query specific
+path_query_matcher = lambda do |request_1, request_2|
```

Comment:
**[DRY]** This method seems to be repeated across different files, are they both necessary or can they be DRYied?